whirlpool/eventsocket.py

```diff
@@ -14,20 +14,42 @@
 
 RECV_MSG_MATCHER = re.compile("{(.*)}\\x00")
 
+# closing frame status codes for websocket.
```

Comment:
```suggestion
```
Instead of the comment, prefix the codes with WS_, for example WS_STATUS_GOING_AWAY.